symfony/plugins/orangehrmPimPlugin/Service/EmployeeService.php

```diff
@@ -293,4 +293,12 @@ public function getEmployeeAsArray(int $empNumber): ?array
         }
         return $this->getNormalizerService()->normalize(EmployeeModel::class, $employee);
     }
+
+    /**
+     * @return array|null
+     */
+    public function getEmailList(): ?array
```

Comment:
no need add method to service, can call through service. e.g. `$this->getEmployeeService()->getEmployeeDao()->getEmailList()`

Reply:
removed from service